src/main/java/kude/Duke.java

```diff
@@ -0,0 +1,14 @@
+package kude;
+
+import kude.tui.Processor;
+import kude.tui.Ui;
+
+public class Duke {
+
+    public static void main(String[] args) {
```

Comment:
Very well organised and clean main method!

Reply:
Thanks :)